pkg/logger/logger.go

```diff
@@ -568,3 +568,28 @@ func GetLevel() Level {
 func HasDebugLevel() bool {
 	return pkgLogger.cfg.Level == DebugLevel
 }
+
+// ErrorFuncName displays a given error prefixed by the current (caller) fn name
+// and a given string in a given format (just like fmt.Sprintf)
+func Errorf(format string, a ...interface{}) error {
```

Comment:
comment with different name

Reply:
I worry about name confusion with (proper) logger function like:

`func Error(msg string, keysAndValues ...interface{}) {`

In the future, we might want to provide, I don't know, `Errorf()` that logs using formats (verbs) instead of key/value pairs. Zap already provides these signatures, we're just ignoring them for now.

How about calling it `NewErrorf()`?

Reply:
Sounds good to me. I'm fixing the testing as well.